torch/csrc/distributed/c10d/init.cpp

```diff
@@ -974,15 +974,48 @@ that adds a prefix to each key inserted to the store.
 
   shared_ptr_class_<::c10d::ProcessGroup::Work>(module, "Work")
       .def("is_completed", &::c10d::ProcessGroup::Work::isCompleted)
```

Comment:
shall we deprecate this as well? This is called `done` in future, see https://github.com/pytorch/pytorch/blob/a3caa719afc3a6fe17c086cde6d5b4d6837ab96c/torch/csrc/jit/python/init.cpp#L1094

Reply:
done

Reply:
This is risky, as we did expose this API in our API doc. If you wanna change this, please mention it where it is defined in `https://github.com/pytorch/pytorch/blob/master/docs/source/distributed.rst` as well.

Reply:
undone